### PR TITLE
Remove use of cookies (Fixes #56)

### LINKED ISSUE #56: Remove use of cookies
As part of our upcoming privacy consent work, all non-essential cookies will now require explicit consent to be granted before they can be set. This presents us with the following options for Traffic Cop (TC):

1. Only initialize experiments if explicit consent has previously been given.
2. Remove reliance on cookies from TC entirely, so it can run independently of consent.

I'm proposing we do 2) because:

1. TC redirects happen before page load, and therefore before consent is likely to have been given previously. Doing 1) will likely result in a considerably number of people who DO give consent, but never actually enter into an experiment because they don't make a repeat visit.
2. TC doesn't actually record any experiment specific data itself. Instead it relies on our regular analytics tools which will require consent separately. 
3. TC does set a cookie to remember which variation a visitor has previously seen, and additionally sets another cookie to record the original referrer (if present).

The cookies TC does set are certainly useful, but given our constraints I think we're likely to get more overall experiment data if we drop them, so TC can run independently of any consent dialog?

The downside is a visitor might see a different experiment variation if they make a return visit during the experiment window. We would also lose the original referrer in our data during the course of an experiment running.

### PR DESCRIPTION
**Pull request by alexgibson**
# Description

This PR removes the use of all cookie from traffic cop, as well as the following dependencies:

- https://github.com/mozmeao/cookie-helper/
- https://github.com/mozmeao/dnt-helper/

I've also made an effort to update the documentation to reflect these changes.

## Testing instructions

1. `npm install`
2. `cd demo && npm install`
3. `npm start`
4. Open http://localhost:3030

#### Redirects

- [x] Clicking the `Page 1 Redirect` link should redirect you to `/page/1` with an experiment variation param (e.g `/page/1?v=4`).
- [x] Refreshing the page should result in you seeing the same experiment variation as above (unless you got no-variation).
- [x] Clicking the `Page 1 Redirect` link multiple times should result in seeing random (different) variations each time.

#### Callbacks

- [x] Clicking the `Page 2 Callback` link should redirect you to `/page/2` with a random taco message.
- [x] Clicking refresh should see a different random taco message each time.

#### Redirects and callbacks

- [x] Clicking the `Page 3 Redirect and Callback` link should see you redirected to `/page/3` with a random experiment variation param (unless you got no-variation), as well as a random custom callback message.
- [x] Refrshing the page should retain the previous variation query param, but the custom callback message should randomly change.
- [x] Clicking the `Page 3 Redirect and Callback` link multiple times should result in random variation query params and callback messages each time.

### CODE REVIEW

File: demo/package.json [hunk 1]
```diff
@@ -4,22 +4,17 @@
     "description": "Express app demonstrating usage of Traffic Cop.",
     "main": "index.js",
     "scripts": {
```
Comment (stephaniehobson) [inline demo/package.json:6]: **question:** Were the webpack changes intentional? I wasn't able to get it to build from the demo folder.

File: src/mozilla-traffic-cop.js [hunk 2]
```diff
@@ -275,42 +161,30 @@ TrafficCop.isRedirectVariation = function (variations, queryString) {
 /*
  * Returns the variation chosen for the current user/experiment.
  */
-TrafficCop.chooseVariation = function (id, variations, totalPercentage) {
-    let rando;
+TrafficCop.chooseVariation = function (variations, totalPercentage) {
+    let random;
```
Comment (stephaniehobson) [inline src/mozilla-traffic-cop.js:165]: **nit-praise**: nice tweak

**Review by stephaniehobson — APPROVED** [review]
Works as expected. I can't check the boxes but they all passed.

Some suggestions and a question about the build process but no blockers.

R+ 👮

File: demo/package.json (hunk 1 above)
Comment (alexgibson) [reply demo/package.json:6]: Ah sorry, no. I forgot to include in the testing instructions that you need to run `npm install && npm run build` first in the root directory before starting up the demo 😶‍🌫️ 

Removing webpack from the demo should be fine here, as all it was doing before this change was importing the cookie helper dependency.

**Pull request merged**